### ISSUE TITLE
std/math_mod macro

### ISSUE THREAD

**Issue by Woccz** [dbaumgarten/yodk#54]
Dependent on #50.

I think it would be good to define a `mod(x,m)` macro in the standard math library.

I've implemented the mod for my own project.

In `std/math_basic.nolol`
```
macro mod(x,m) expr
    x-x/m/1000*1000*m
end
```
In `std/math_advanced.nolol`
```
macro mod(x,m) expr
    x%m 
end
```
(Macro definitions assuming #55.)

Thank you.

**Comment by dbaumgarten** [comment]
Seems reasonable. In the long run, once #50 is done, there should be replacement-macros for all build-in chip operators (except for these that are available on all chips).

**Comment by dbaumgarten** [comment]
Done. Published as part of v0.1.5
